app/controllers/publish/courses/age_range_controller.rb

```diff
@@ -14,7 +14,7 @@ def edit
 
       def update
         if form_object.valid?
-          flash[:success] = I18n.t("success.saved")
+          flash[:success] = @course.is_published? ? I18n.t("success.value_published", value: "age range") : I18n.t("success.value_saved", value: "age range")
```

Comment:
This is used in a few places, is it worth putting into a method/concern which other controllers can re-use? then each one would just call the method with the value?

Reply:
Done